go-controller/pkg/node/helper_linux.go

```diff
@@ -54,3 +56,22 @@ func getIntfName(gatewayIntf string) (string, error) {
 	}
 	return intfName, nil
 }
+
+func deleteConntrack(ip string) {
+	ipAddress := net.ParseIP(ip)
+	if ipAddress.To4() == nil {
+		return
+	}
+	filterTypes := []netlink.ConntrackFilterType{netlink.ConntrackOrigSrcIP, netlink.ConntrackOrigDstIP, netlink.ConntrackReplySrcIP, netlink.ConntrackReplyDstIP, netlink.ConntrackReplyAnyIP}
```

Comment:
this function is almost identical to deletePodConntrack. You could just call this function in deletePodConntrack, other than the fact that the filter type here has extra criteria (this line is also too long). The extra criteria here is:
netlink.ConntrackReplySrcIP, netlink.ConntrackReplyDstIP

Why do you include those here, but not in the deletePodConntrack filter? It should be OK to include it in the pod filter as well I think, as it would just remove any conntrack entries for the pod where the DNAT is happening to the pod ip, but it's hard to know without looking specifically at the entries.

Reply:
I changed the location of the function to pkg/util/net_linux.go and made wrappers in the other linux files to call it 

done